tui/src/state.rs

```diff
@@ -24,7 +24,7 @@ use temp_dir::TempDir;
 
 const MIN_WIDTH: u16 = 77;
 const MIN_HEIGHT: u16 = 19;
-const TITLE: &str = concat!("Linux Toolbox - ", env!("BUILD_DATE"));
+const TITLE: &str = concat!("Linux Toolbox - ", env!("VERSION"));
```

Comment:
https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-crates

```suggestion
const TITLE: &str = concat!("Linux Toolbox - ", env!("CARGO_PKG_VERSION"));
```